src/pages/Appointments/components/PractitionerSelector.tsx

```
@@ -493,45 +516,66 @@ export const PractitionerSelector = ({
+              {/* Departments Section */}
+              {filteredChildOrganizations.length > 0 && (
+                <div className="space-y-1">
+                  <h3 className="px-2 py-1 text-xs font-medium text-gray-500 uppercase tracking-wide">
+                    {t("departments")}
+                  </h3>
+                  {filteredChildOrganizations.map((organization) => (
+                    <div
+                      key={organization.id}
+                      className="flex items-center gap-3 p-3 rounded-lg hover:bg-gray-50 cursor-pointer"
+                      onClick={() => handleChildOrganizationClick(organization)}
+                    >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)